controller/src/settings/config.rs

```diff
@@ -8,6 +8,7 @@ use std::{
 };
 
 use anyhow::Context;
+use cs2_schema_generated::cs2::client::C_EnvVolumetricFogController;
```

Comment:
Hui, how did this ended up here :D

Reply:
i don't know either, it was probably the rust helper in visual studio code